data/test/_main.cfg

```diff
@@ -81,6 +81,7 @@
 {test/scenarios/wml_tests/UnitsWML/AbilitiesWML/heals}
 {test/scenarios/wml_tests/UnitsWML/AbilitiesWML/regenerate}
 {test/scenarios/wml_tests/UnitsWML/AbilitiesWML/resistance}
+{test/scenarios/wml_tests/UnitsWML/AbilitiesWML/leadership}
```

Comment:
Please put it in the alphabetically-sorted place

Reply:
Updated.